src/common/api/utils.ts

```diff
@@ -24,3 +24,11 @@ export const getNetworkModeFromNetworkId = (networkId: ChainID) => {
       return undefined;
   }
 };
+
+export const constructQueryParams = (obj: Record<string, string | number | boolean>): string => {
```

Comment:
I don't see this function called here, is it needed?

Reply:
I was using it before but this is not used anymore. I will remove it.